packages/clay-shared/README.md

```diff
@@ -0,0 +1,17 @@
+# clay-shared
+
+Shared utilities and hooks used across clay components
+
```

Comment:
hey @bryceosterhaus you can put some warnings that this package for a moment is only for internal use and that the API's are unstable and can be modified at any time and that "use at your own risk".

Reply:
Good call, I went ahead and updated the README.